src/lib/render.py

```diff
@@ -44,7 +44,31 @@ def create_figure(image_data, wcs, colour_map, vmin, vmax, s, contour_levels):
     return fig, image
 
 
-def update_image_norm(image, image_data, vmin, vmax, s):
+def create_figure_png(image_data):
+    fig = Figure(figsize=(5, 5), dpi=150)
+    fig.patch.set_facecolor("#afbac5")
+    ax = fig.add_subplot()
+    fig.subplots_adjust(top=0.95, bottom=0.2, right=0.95, left=0.2, hspace=0, wspace=0)
+    # ax.tick_params(axis="both", which="major", labelsize=5)
```

Comment:
Wondering if we ever got confirmation that we shouldn't have ticks for pixel values, if so I gues can just remove this. but it's a nitpick

Reply:
I'll remove it, can always add it back if we need them